README.md

```diff
@@ -86,11 +86,13 @@ _Alternatively, the `deploy-bundles.sh` script can be used to ensure only un-pub
 
 
 ### 1.a. Verify Bundle Deployment
+
 ```zsh
 # returns a global list of deployed bundles
-agd query vstorage children swingStore.bundle --node $NODE
-# query for a specific bundle
-agd query vstorage data swingStore.bundle.[bundle id] --node $NODE
+agd query vstorage data bundles --node $NODE --chain-id $CHAIN_ID --output json
+# returns a formattted list of deployed bundles
```

Comment:
I noticed `swingStore.bundle` is gone! On devnet, I see just one bundle deployed (probably khread), and am unsure how to verify the inter bundles.

Reply:
> I noticed `swingStore.bundle` is gone.

Yeah; I learned after we found it that it was a transitionary thing.

> For now, i am looking at the `bundles` table

That's the right place to look, though of course it requires running your own node.